### PR TITLE
OCPBUGS#10586: Removed Dev Preview note for installing on OCI

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/OSDOCS-10586

Link to docs preview: https://76852--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-oci

Review:
- [x] QE has approved this change.
- [x] PM has approved this change.
- [x]  SME has approved this change.

### CODE REVIEW

**Comment by makentenza** [comment]
LGTM

**Review by makentenza — APPROVED** [review]
LGTM

**Comment by mhanss** [comment]
LGTM

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -953,14 +953,10 @@ For more information, see xref:../storage/container_storage_interface/persistent
 [id="ocp-4-14-oci"]
 === Oracle(R) Cloud Infrastructure
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-14-release-notes.adoc:954]: WDYT? I see this style adopted to put emphasis on the note as the TP table entry is elsewhere in the doc.

[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-14-release-notes.adoc:954]: Then again, I see that https://github.com/openshift/openshift-docs/pull/76850 is removing the TP, so all references to dev preview should be removed for this feature? I ask as the TP tables have an entry that needs updating.

**Review by makentenza — APPROVED** [review]
LGTM

**Pull request merged**